src/services/api/index.ts

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "http://localhost:3333",
```

Comment:
it is interesting that this value is in an environment variable

Reply:
Doing that posthaste